packages/entity-browser/src/components/FormField/formFieldUtil.js

```diff
@@ -0,0 +1,98 @@
+  'ch.tocco.nice2.model.form.components.simple.MultiSelectBox': 'multi-select',
+  'ch.tocco.nice2.model.form.components.simple.SingleSelectBox': 'single-select',
+  'ch.tocco.nice2.model.form.components.simple.RemoteField': '',
+  'ch.tocco.nice2.model.form.components.simple.MultiRemoteField': '',
+  'ch.tocco.nice2.model.form.components.simple.ListPanel': '',
+  'ch.tocco.nice2.model.form.components.simple.ReferencesListPanel': '',
+  'ch.tocco.nice2.model.form.components.simple.PasswordField': '',
+  'ch.tocco.nice2.model.form.components.simple.CreatePasswordField': '',
+  'ch.tocco.nice2.model.form.components.simple.RangeField': '',
+  'ch.tocco.nice2.model.form.components.simple.DisplayField': '',
+  'ch.tocco.nice2.model.form.components.simple.DisplayExpressionFielxpressionFieldFacade': '',
+  'ch.tocco.nice2.model.form.components.simple.MoneyAmountField': '',
+  'ch.tocco.nice2.model.form.components.simple.PercentField': '',
+  'ch.tocco.nice2.model.form.components.simple.DataAmountField': '',
+  'ch.tocco.nice2.model.form.components.simple.CustomDataField': '',
+  'ch.tocco.nice2.model.form.components.simple.PhoneField': 'phone',
+  'ch.tocco.nice2.model.form.components.table.Table': '',
+  'ch.tocco.nice2.model.form.components.table.Column': '',
+  'ch.tocco.nice2.model.form.components.action.Action': '',
+  'ch.tocco.nice2.model.form.components.action.ActionSeparator': '',
+  'ch.tocco.nice2.model.form.components.Form': '',
+  'ch.tocco.nice2.model.form.components.Template': '',
+  'ch.tocco.nice2.model.form.components.layout.HorizontalBox': '',
+  'ch.tocco.nice2.model.form.components.layout.VerticalBox': '',
+  'ch.tocco.nice2.model.form.components.navigation.IteratorComponent': '',
+  'ch.tocco.nice2.model.form.components.simple.DescriptionField': '',
+  'ch.tocco.nice2.model.form.components.composite.LocationField': '',
+  'ch.tocco.nice2.model.form.components.simple.LoginField': ''
+}
+
+const getOptions = (formField, modelField, util) => {
+  const options = {}
+  switch (formField.type) {
+    case 'ch.tocco.nice2.model.form.components.simple.MultiSelectBox':
+    case 'ch.tocco.nice2.model.form.components.simple.SingleSelectBox':
+      if (util.relationEntities) {
+        const fieldStore = util.relationEntities[modelField.targetEntity]
+        options.store = fieldStore ? fieldStore.data : []
+      }
+  }
+
+  return options
+}
+
+const getEvents = (field, modelField, util) => {
+  const events = {}
+  switch (field.type) {
+    case 'ch.tocco.nice2.model.form.components.simple.MultiSelectBox':
+    case 'ch.tocco.nice2.model.form.components.simple.SingleSelectBox':
+      if (util.loadRelationEntity) {
+        events.onFocus = () => {
+          util.loadRelationEntity(modelField.targetEntity)
+        }
+      }
+  }
+
+  return events
+}
+
+export const getEditableValueProps = (formField, modelField, util) => {
+  const type = fromDefinitionTypeMap[formField.type]
```

Comment:
maybe we should throw or log an error if no `type` could be found for `formField.type` to make it easier to find the issue if a new component type has been added. could be hard to find if `undefined` is returned without logging a warning.